frontend/src/pages/articles/components/article-card/article-card.tsx

```diff
@@ -1,71 +1,31 @@
 import ArticlePreview from '~/assets/img/article-preview.png';
 import { Avatar, Icon, Link, Tag } from '~/libs/components/components.js';
 import { AppRoute } from '~/libs/enums/enums.js';
+import { type UserAuthResponseDto } from '~/packages/users/users.js';
+import { type ArticleType } from '~/pages/articles/libs/types/types.js';
 
-import {
-  type ArticleType,
-  type TagType,
-  type UserAuthResponseDto,
-} from './libs/types/types.js';
+import { Reaction } from '../components.js';
+import { REACTIONS_LIST } from './libs/constants.js';
 import styles from './styles.module.scss';
 
 type Properties = {
   article: ArticleType;
   user: UserAuthResponseDto;
 };
 
-const MOCKED_TEXT: string = `The developer technology landscape changes all the time as new tools
-  and technologies are introduced. After having lots of interviews and
-  reading through countless job descriptions on job boards I think this
-  is a great modern tech stack for JavaScript developers in 2021.`;
-
-const MOCKED_TAGS: TagType[] = [
-  { id: 1, name: 'IT' },
-  { id: 2, name: 'CODE' },
-  { id: 3, name: 'Humor' },
-  { id: 4, name: 'Work' },
-  { id: 5, name: 'Tech' },
-];
-
-const MOCKED_ARTICLE: ArticleType = {
-  publishedAt: 'May 28',
-  timeSincePublication: '7 min read',
-  title: 'Modern Full-Stack Developer Tech Stack 2021',
-  text: MOCKED_TEXT,
-  comments: '540',
-  views: '367K',
-  likes: '36K',
-  dislikes: '18K',
-  tags: MOCKED_TAGS,
-};
-
-const MOCKED_USER: UserAuthResponseDto = {
-  id: 7,
-  email: 'nolanaris@gmail.com',
-  firstName: 'Nolan',
-  lastName: 'Saris',
-};
-
-const ArticleCard: React.FC<Properties> = ({
-  article = MOCKED_ARTICLE,
-  user = MOCKED_USER,
-}) => {
-  const {
-    publishedAt = 'May 28',
-    timeSincePublication = '7 min read',
-    title = 'Modern Full-Stack Developer Tech Stack 2021',
-    text = MOCKED_TEXT,
-    comments = '540',
-    views = '367K',
-    likes = '36K',
-    dislikes = '18K',
-    tags: propertyTags,
-  } = article;
+const ArticleCard: React.FC<Properties> = ({ article, user }) => {
+  const { publishedAt, timeSincePublication, title, text, tags } = article;
 
   const { firstName, lastName } = user;
   const fullName = `${firstName} ${lastName}`;
-  const tags = propertyTags.map((tag) => <Tag key={tag.id} name={tag.name} />);
-
+  const renderTags = tags.map((tag) => <Tag key={tag.id} name={tag.name} />);
+  const renderReactions = REACTIONS_LIST.map((reaction) => (
+    <Reaction
+      key={reaction.reaction}
+      iconName={reaction.iconName}
+      reactions={article[reaction.reaction]}
+    />
+  ));
```

Comment:
I believe Nazar meant to use ul and list items per different staff
Also it's not very clear how it looks for now - make them render separately by in `<ul><li>...</ul>` form
Same for `tags`

Reply:
Can I create common tags component instead of tag one?
Or would it be better to remove the tag component when the article view task is merged, since that component is used there

Reply:
Just use `<ul><li>...</ul>` as later on we'd make some of them interactive (likes/dislikes/comments) and they'd have separate logic -- there is no need to create something common